include/eve/detail/function/simd/common/combine.hpp

```diff
@@ -34,6 +34,13 @@ namespace eve::detail
 
       return that;
     }
+    else if constexpr( is_bundle_v<abi_t<T, N>> )
+    {
+      return that_t ( kumi::map ( [](auto ml, auto mh) { return eve::combine(ml,mh); }
```

Comment:
I think you can just pass `eve::combine` here.

Reply:
Right, probably wrote that before adapting combine